joplin/static/js/admin.js

```diff
@@ -1,12 +1,55 @@
-document.addEventListener('DOMContentLoaded', function(event) {
+// TODO: This a better way
+const anchorLinks = {
+  id_title:
+    "https://briaguya.github.io/digital-services-style-guide/writing-service-pages/#title",
+  id_steps:
+    "https://briaguya.github.io/digital-services-style-guide/writing-service-pages/#steps",
+  id_apps:
+    "https://briaguya.github.io/digital-services-style-guide/writing-service-pages/#apps",
+  id_additional_content:
+    "https://briaguya.github.io/digital-services-style-guide/writing-service-pages/#additional",
+  id_contacts:
+    "https://briaguya.github.io/digital-services-style-guide/writing-service-pages/#contacts"
+};
+
+document.addEventListener("DOMContentLoaded", function(event) {
   // HACK: I can't find a way to override this in python
-  const title = document.querySelector('label[for=id_title]');
-  if (title) {
-    title.textContent = 'Actionable Title';
+  // Get all labels and turn them into links
+  const labels = document.querySelectorAll("label");
+
+  for (const label of labels) {
+    let id = label.getAttribute("for");
+    if (!id) {
+      // HACK: Only some fields actually have for attributes
+      switch (label.innerText) {
+        case "ADD ANY MAPS OR APPS THAT WILL HELP THE RESIDENT USE THE SERVICE ":
+          id = "id_apps";
+          break;
+        case "CONTACTS":
+          id = "id_contacts";
+          break;
+      }
+      debugger;
+    }
+
+    if (id in anchorLinks) {
+      text = document.createTextNode(label.textContent);
+      label.textContent = "";
+      if (id === "id_title") {
+        text = document.createTextNode("Actionable Title");
```

Comment:
This if statement is a little smelly. Why is there this one-off?

Reply:
This was a one off hack that was already in the codebase:
```
// HACK: I can't find a way to override this in python
const title = document.querySelector('label[for=id_title]');
if (title) {
  const labels = document.querySelectorAll("label");
  title.textContent = 'Actionable Title';
```